src/main/resources/db/changesets/20200625_initial_schema.yaml

```diff
@@ -0,0 +1,143 @@
+# NOTE: this is still changing because nothing uses Janitor. Expect
+# DBs to keep blowing up for now.
+databaseChangeLog:
+- changeSet:
+    id: init_janitor_table
+    author: yonghao
+    changes:
+    - createTable:
+        tableName: resource_type
+        columns:
+          - column:
+              name: type_id
```

Comment:
changed for resource_type and tracked_resource table.
For gcp_bucket, cleanup_log, label table, still use tracked_id as I feel that might be better since whose are the FK from tracked_resource, and tracker_id is easier to understand?